e2e-tests/tests/committee/conftest.py

```diff
@@ -441,14 +442,21 @@ def _inner(mc_epoch):
                 probability_of_selection_in_single_epoch = (permissioned_seats / total_committee_seats) * (
                     PERMISSIONED_CANDIDATE_WEIGHT / permissioned_candidates_count
                 )
+                expected_attendance = permissioned_seats * (
+                    PERMISSIONED_CANDIDATE_WEIGHT / permissioned_candidates_count
```

Comment:
IMO `PERMISSIONED_CANDIDATE_WEIGHT` should just disappear. It is one, the neutral element of multiplication used in formulas that are only multiplications and divisions.

What is more:
```
probability_of_selection_in_single_epoch = (permissioned_seats / total_committee_seats) * (
                    PERMISSIONED_CANDIDATE_WEIGHT / permissioned_candidates_count
                )
```
is now invalid. If P=1, R=2, and there is 1 permissioned candidate, his probability is 1, but according to this formula it would be `(1/3) * (1/1)`, which is `1/3`.